tests/TestHelpers/GraphHelper.php

```diff
@@ -483,7 +483,7 @@ public static function deleteGroup(
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public static function addUsersToGroup(
+	public static function addUsersOrGroupsToGroup(
```

Comment:
can we also add groups to group?

Reply:
this method is created to handle negative testing https://github.com/owncloud/ocis/pull/5693/files#diff-11797d1463e39737e8ebc0021e068a2b20385bae47a266a4a389b873887c197cR230

Reply:
`addUsersToGroup` is fine here. Since we cannot add groups to a group, `addUsersOrGroupsToGroup` is not suitable.

Reply:
then how to handle adding groups to group api request